core/core-dotspacemacs.el

```diff
@@ -126,8 +126,11 @@ Default value is `cache'.")
   "If non nil the paste micro-state is enabled. While enabled pressing `p`
 several times cycle between the kill ring content.'")
 
-(defvar dotspacemacs-guide-key-delay 0.4
-  "Guide-key delay in seconds.")
+(defvar dotspacemacs-which-key-delay 0.4
+  "Which-key delay in seconds.")
```

Comment:
Maybe more helpful: "Which-key appears this many seconds after the last key stroke."